zerver/tests/fixtures/markdown_test_cases.json

```diff
@@ -1281,9 +1276,9 @@
       "http://foo.com/blah_(wikipedia)_blah#cite-1"
     ],
     [
-      "http://technet.microsoft.com/en-us/library/Cc751099.rk20_25_big(l=en-us).mov",
-      "<p>%s</p>",
-      "http://technet.microsoft.com/en-us/library/Cc751099.rk20_25_big(l=en-us).mov"
+      "https://en.wikipedia.org/wiki/File:Methamphetamine_from_ephedrine_with_HI_en.mov",
+      "<div class=\"message_inline_video\"><video class=\"inline_video\" controls=\"true\" preload=\"metadata\" src=\"https://external-content.zulipcdn.net/external_content/dc9d11c74b0c68cd2084eaa38e36f7fac3883f37/68747470733a2f2f656e2e77696b6970656469612e6f72672f77696b692f46696c653a4d657468616d70686574616d696e655f66726f6d5f657068656472696e655f776974685f48495f656e2e6d6f76\"></video></div>",
+      "https://en.wikipedia.org/wiki/File:Methamphetamine_from_ephedrine_with_HI_en.mov"
```

Comment:
This belongs more in `regular_tests`, with the other embedding tests (e.g. for images) than it does here with the linkifiers.

Reply:
Oh, didn't realize that.